workflow/km.scpt

```diff
@@ -1,7 +1,7 @@
 try
 	get application id "com.stairways.keyboardmaestro.engine"
 on error err_msg number err_num
-	return "Keyboard Maestro not found. First install it and then use this workflow."
+	return "Keyboard Maestro not found. You should install it first."
```

Comment:
I display this error message in Alfred. Previously it was too long, so it was truncated. I shortened it a bit, so it can be displayed fully.